deploy/files/conf/crontab

```diff
@@ -2,8 +2,6 @@
 # HOWEVER: these commands will NOT run on instances booted from the EE AMI
 # If you want to have a command run only once per env / once globally, use SSM Run Command
 MAILTO=developers@democracyclub.org.uk
-# Import councils from EC hourly
-1 * * * * polling_stations /usr/bin/manage-py-command import_councils --only-contact-details --database default
```

Comment:
As discussed, I've just disabled this job that is running against the replicas.
We've got a scheduled job to do this on the primary
https://github.com/DemocracyClub/UK-Polling-Stations/blob/3a0bb9dff904bf79030df4492f2148e04ef75a65/cdk/stacks/command_runner_stack.py#L50-L54
and we replicate from there